### ISSUE TITLE
rocket-info.lua attempt to call field 'destory_if_valid' (a nil value)

### ISSUE THREAD

**Issue by flbuddymooreiv** [explosivegaming/scenario#124]
**Factorio version**
Version number: 0.16.64

**Scenario version**
Version number: 2019/08/13

**Bug description**
```
58287.854 Script @/home/buddy/factorio/factorio_experimental-0.17.56/factorio/temp/currently-playing/utils/event_core.lua:30: ...torio/temp/currently-playing/modules/gui/rocket-info.lua:367: attempt to call field 'destory_if_valid' (a nil value)
```

**Reproduction method**
1. Connect to S3
2. See error in log

**Expected behavior**
There's no real manifested problem - just logs in the error

**Screenshots**
See the log output from above

**Additional context**
Cooldude made me do it (submit the bug report)!

**Comment by flbuddymooreiv** [comment]
Also you may want to replace destory with destroy :)

**Comment by Cooldude2606** [comment]
Fixed in v5.9.1
[Fixed destroy_if_valid for rocket gui](https://github.com/explosivegaming/scenario/commit/b95f6b99c478d8a5b3a04e97547a4b689d1de3ca)
